FLT/Mathlib/RingTheory/Norm/Defs.lean

```diff
@@ -0,0 +1,8 @@
+import Mathlib.RingTheory.Norm.Defs
+
+namespace LinearMap
+variable {M A B : Type*} [CommRing A] [CommRing B] [AddCommGroup M] [Algebra A B] [Module A M]
+  [Module B M] [IsScalarTower A B M] [Module.Finite A B] [Module.Finite B M]
```

Comment:
```suggestion
  [Module B M] [IsScalarTower A B M] [Module.Finite A B] [Module.Finite B M]
  [Module.Free A B] [Module.Free B M]
```
otherwise it's likely to be false (`norm` is pathological without freeness and probably det is too).

Reply:
I have added this file to `main` so I can make an issue of the `sorry`.